src/js/tracing/reactnativetracing.ts

```diff
@@ -182,6 +186,23 @@ export class ReactNativeTracing implements Integration {
 
     this._getCurrentHub = getCurrentHub;
 
+    const clientOptionsTracePropagationTargets = clientOptions && clientOptions.tracePropagationTargets;
+    // There are three ways to configure tracePropagationTargets:
+    // 1. via top level client option `tracePropagationTargets`
+    // 2. via ReactNativeTracing option `tracePropagationTargets`
+    // 3. via ReactNativeTracing option `tracingOrigins` (deprecated)
+    //
+    // To avoid confusion, favour top level client option `tracePropagationTargets`, and fallback to
+    // ReactNativeTracing option `tracePropagationTargets` and then `tracingOrigins` (deprecated).
```

Comment:
Is `tracingOrigins` marked as deprecated in the source code? so users are aware.

Reply:
Yes, it is.
RN shares the options interface with JS.
https://github.com/getsentry/sentry-javascript/blob/eee291b8960d4cb290856ff36a1199599cd11f87/packages/tracing-internal/src/browser/request.ts#L19-L24